node/core/pvf/Cargo.toml

```diff
@@ -42,9 +43,11 @@ substrate-build-script-utils = { git = "https://github.com/paritytech/substrate"
 [dev-dependencies]
 assert_matches = "1.4.0"
 hex-literal = "0.3.4"
+polkadot-node-core-pvf-common = { path = "common", features = ["test-utils"] }
 
 adder = { package = "test-parachain-adder", path = "../../../parachain/test-parachains/adder" }
 halt = { package = "test-parachain-halt", path = "../../../parachain/test-parachains/halt" }
 
 [features]
 ci-only-tests = []
+test-utils = []
```

Comment:
One comment please on how to use it.

Reply:
Ok, I'm going to add. Thanks!

Reply:
@ggwpez Comment added. Is this what you had in mind?